ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Tables/Convenience.hs

```diff
@@ -0,0 +1,186 @@
+{-# LANGUAGE FlexibleContexts    #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeFamilies        #-}
+
+-- |
+
+module Ouroboros.Consensus.Ledger.Tables.Convenience (
```

Comment:
We have a bunch of `*Utils` modules. Maybe we should rename this.

Reply:
Done